app/script_tracking.py

```diff
@@ -0,0 +1,61 @@
+"""Tools for modifying 'bot_scripts' table"""
```

Comment:
With this file you can either pull the latest version from my branch or leave this file out altogether and it will return when I merge my feature branch. Nothing here is going to break the app if missing, just a few testing endpoints won't work until my branch is merged.